content/cli/usages.md

```diff
@@ -1,169 +1,169 @@
-| `--project [project]`           | Project that element should be added to.<br/> Alias: `-p`                                                       |
-| `--flat`                        | Do not generate a folder for the element.                                                                       |
-| `--collection [collectionName]` | Specify schematics collection. Use package name of installed npm package containing schematic.<br/> Alias: `-c` |
-| `--spec`                        | Enforce spec files generation (default)                                                                         |
-| `--no-spec`                     | Disable spec files generation                                                                                   |
+| Option                          | Description                                                                                                                          |
+| ------------------------------- |--------------------------------------------------------------------------------------------------------------------------------------|
+| `--dry-run`                     | Rapporte les modifications qui seraient apportées, mais n'effectue pas de modifications sur le système de fichiers.<br/> Alias: `-d` |
+| `--project [project]`           | Le projet auquel cet élément devrait être ajouté.<br/> Alias: `-p`                                                                   |
+| `--flat`                        | Ne génère pas de dossier pour cet élément.                                                                                           |
+| `--collection [collectionName]` | Spécifie la collection de schémas. Utilise le nom du package de l'installation npm contenant le schéma.<br/> Alias: `-c`             |
+| `--spec`                        | Impose la génération des fichiers de tests (par défaut).                                                                             |
+| `--no-spec`                     | Désactive la génération des fichiers de tests.                                                                                       |
 
 #### nest build
 
-Compiles an application or workspace into an output folder.
+Compile une application ou un espace de travail dans un dossier de sortie.
 
-Also, the `build` command is responsible for:
+La commande `build` est aussi responsable de:
 
-- mapping paths (if using path aliases) via `tsconfig-paths`
-- annotating DTOs with OpenAPI decorators (if `@nestjs/swagger` CLI plugin is enabled)
-- annotating DTOs with GraphQL decorators (if `@nestjs/graphql` CLI plugin is enabled)
+- Mapping des chemins (si l'on utilise des alias de chemins) via `tsconfig-paths`
+- Annotation des DTO avec des décorateurs OpenAPI (si le plugin CLI `@nestjs/swagger` est activé)
+- Annotation des DTO avec des décorateurs GraphQL decorators (si le plugin CLI `@nestjs/graphql` CLI plugin est activé)
```

Comment:
maping, annotating : suite de "is responsible for" donc à traduire par des infinitifs. Responsable de - mapper des chemins... annoter des DTO...